src/routes/LandingPage/components/AuthButton/index.jsx

```diff
@@ -0,0 +1,8 @@
+import { cloneElement } from "react";
+
+const AuthButton = props =>
+  cloneElement(props.children, {
```

Comment:
do we need cloneElement here?

Reply:
It's probably used to pass props to children.
Maybe a cleaner approach would be to use render props?
https://reactjs.org/docs/render-props.html